tests/SpecTests/TransactionsSpecTest.php

```diff
@@ -214,7 +218,7 @@ protected function createTestCollection()
         $context = $this->getContext();
 
         $database = $context->getDatabase();
-        $database->createCollection($context->collectionName, $context->defaultWriteOptions);
+        $database->createCollection($context->collectionName, ['writeConcern' => new WriteConcern('majority')] + $context->defaultWriteOptions);
```

Comment:
Why was this necessary? Doesn't `Context::fromTransactions()` already apply `w:majority` for the default WC options?

Reply:
I first ran into test failures and added this to avoid issues but forgot to remove all instances of that. Thanks for pointing it out 👍